mms/tests/unit_tests/test_otf_codec_protocol.py

```diff
@@ -0,0 +1,298 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""
+On The Fly Codec tester
+"""
+
+from collections import namedtuple
+
+import pytest
+
+from mms.mxnet_model_service_error import MMSError
+from mms.utils.model_server_error_codes import ModelServerErrorCodes as Err
+from mock import Mock
+from mms.protocol.otf_message_handler import OtfCodecHandler
+
+
+@pytest.fixture()
+def socket_patches(mocker):
+    Patches = namedtuple('Patches', ['socket'])
+    mock_patch = Patches(mocker.patch('socket.socket'))
+    mock_patch.socket.recv.return_value = b'1'
+    return mock_patch
+
+
+@pytest.fixture()
+def get_otf_codec_fixture():
```

Comment:
Rename this function to just oft_codec and directly "return OftCodeHandler()"

Reply:
Good catch of returning the OtfCodec directly. Had done this for some debugging.